### DISCUSSION TITLE
Changing resolution on start

### DISCUSSION THREAD

**Discussion by KatyPurry** [LizardByte/community#181] [Sunshine] [answered]
Hello,
Currently I'm using Multimonitor.exe with 2 .bat files to change my resolution before launching Sunshine. Mainly because my iPad is in 4:3 and my host screen is in 16:10.

It works but it forces me to spend time clicking on the bat files to change my resolutions. Is there a method with Sunshine to for example launch a bat file at Sunshine launch and execute the second one when I quit the program?

I know GameStream Launchpad but i'm on AMD (6800XT), so it's unavailable for me.

Thank you !

**Comment by KuleRucket** [comment]
https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/usage.html#application-list

**Comment by KatyPurry** [comment]
Thank you, I'll check (I'm not a developper at all), I will take the liberty of posting a new message if I don't understand everything.

**Comment by KuleRucket** [comment]
You can configure it all in the web UI under "Applications". The documents just give extra info

**Reply by KatyPurry** [reply]
Thank you,



In Desktop on the "do" field, I added the content of my bat file : `start "" "C:\MultiMonitorTool.exe" /SetMonitors "Name=\\.\DISPLAY2 Width=1600 Height=1200"`
And for "undo" : `start "" "C:\MultiMonitorTool.exe" /SetMonitors "Name=\\.\DISPLAY2 Width=2560 Height=1440"`

But it returns an error in the log : `[2023:02:22:23:55:21]: Fatal: C:\Users\alexa\OneDrive\Documents\Sunshine\config/apps.json`

![image](https://user-images.githubusercontent.com/89610869/220788589-b7b7a9a5-68f9-4dae-af94-94c1f13af813.png)

**Comment by KatyPurry** [comment]
After a few days of trying, I still can't get these commands to work. Do you have any idea how I can get out of this?

**Comment by Puni123** [comment]
Maye this works as an example - works for me for steam - but you need the qres.exe program. (took it from a thread from skellycode.

```json
{
    "env": {
        "PATH": "$(PATH);C:\\Program Files (x86)\\Steam"
    },
    "apps": [
        {
            "name": "Steam Big Picture",
            "output": "steam.txt",
            "detached": [
                "steam steam:\/\/open\/bigpicture"
            ],
            "image-path": "steam.png",
            "prep-cmd": [
                {
                    "do": "C:\\Users\\USERNAME\\QRes.exe \/x 3840 \/y 2160",
                    "undo": "C:\\Users\\USERNAME\\QRes.exe \/x 3440 \/y 1440"
                }
            ]
        }
    ]
}
```

**Comment by ReenigneArcher** [comment, marked as answer]
There's updated instructions in our nightly docs, they'll be in latest after next release. https://docs.lizardbyte.dev/projects/sunshine/en/nightly/about/app_examples.html

**Reply by Puni123** [reply]
great - thx!

**Comment by KatyPurry** [comment]
Awesome thank you @Puni123 @ReenigneArcher
